in_toto/exceptions.py

```diff
@@ -39,5 +39,5 @@ class PrefixError(Error):
     """Indicates that there is an error because of the prefixes passed."""
 
 
-class InvalidMetadata(Error):
+class InvalidMetadata(Error):  # noqa: N818
```

Comment:
The "proper" fix would break backward compatibility:
```suggestion
class InvalidMetadataError(Error):
```